src/unix.rs

```diff
@@ -202,22 +249,31 @@ impl Client {
     }
 
     pub fn string_arg(&self) -> String {
-        format!("{},{}", self.read.as_raw_fd(), self.write.as_raw_fd())
+        match self {
+            Client::Pipe { read, write } => format!("{},{}", read.as_raw_fd(), write.as_raw_fd()),
+            Client::Fifo { path, .. } => format!("fifo:{}", path.to_str().unwrap()),
```

Comment:
I think it's a bad idea to use the new `fifo:` syntax by default since this is a new syntax and not every program supports this.

Reply:
Correct. That's why we only support this syntax when inheriting from the environment, and it's really in a fifo style.

Reply:
> Correct. That's why we only support this syntax when inheriting from the environment, and it's really in a fifo style.

Yeah this makes sense, though IMO it would be better to keep using the old syntax.

I do appreciate this new syntax since it means we no longer have to register a callback to `Command::pre_exec`, which blocks the use of `vfork`.